lib/filter-overlapping-peaks.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * This function de-dups overlapping peaks in each dataset from ChIP, DNase and ATAC-seq assays.
+ * @param {Array} Datasets list of datasets to filter
+ * @returns {Array} Return an array. Array[0] si DatasetsFiltered: datasets after filtering. Array[1] is DatasetsFilteredCount: the number of datasets that are filtered out.
+ */
+export default function filterOverlappingPeaks(Datasets) {
+  const DatasetsFiltered = [];
+  let DatasetsFilteredCount = 0;
+  if (Datasets.length > 0) {
+    // we want to keep the wider peaks in each dataset
+    // if the start & end positions are the same we will keep the one with the strongest signal
+    // sort all peaks by dataset ids -> start positions in ascending order -> end positions in descending order -> signals in descending order
+    Datasets.sort(
```

Comment:
You have the lodash npm package installed, and it has a [sortBy](https://lodash.com/docs/4.17.15#sortBy) function that’s typically easier to use. Not sure though if that `localCompare` prohibits using `sortBy` or not, since I don’t know what that does. Also `sortBy` is non-mutating. Is it odd that `Datasets` is uppercase?